cni/network/network.go

```diff
@@ -1143,7 +1143,14 @@ func determineSnat() (bool, bool, error) {
 		return snatConfig.EnableSnatForDns, snatConfig.EnableSnatOnHost, retrieveSnatConfigErr
 	}
 
-	log.Printf("[cni-net] EnableSnatOnHost set to %t; EnableSnatForDns set to %t", snatConfig.EnableSnatOnHost, snatConfig.EnableSnatForDns)
+	log.Printf("[cni-net] saved snat settings %+v to %s", snatConfig, snatConfigFile)
+	if snatConfig.EnableSnatOnHost && snatConfig.EnableSnatForDns {
+		log.Printf("[cni-net] enabling SNAT on container host for outbound connectivity")
+	} else if snatConfig.EnableSnatForDns && !snatConfig.EnableSnatOnHost {
+		log.Printf("[cni-net] enabling SNAT on container host for DNS traffic")
```

Comment:
checking it enable for snat for DNS should suffice - other should be implicit?

Reply:
fixed